src/main/java/com/example/auctionapp/enumeration/SizeEnum.java

```diff
@@ -0,0 +1,23 @@
+package com.example.auctionapp.enumeration;
+
+public enum SizeEnum {
+    EXTRA_SMALL ("Extra Small"),
+    SMALL ("Small"),
+    MEDIUM ("Medium"),
+    LARGE ("Large"),
+    EXTRA_LARGE ("Extra Large");
+
+    public final String label;
+
+    private SizeEnum(String label) {
+        this.label = label;
+    }
+
+    public static SizeEnum fromValue(String size) {
+
+            if(size.contains(" ")) {
+                size = size.replace(" ", "_");
+            }
+        return SizeEnum.valueOf(size.toUpperCase());
```

Comment:
Nope, this is not the way to do this. `fromValue` means constructing the enum from the value it holds. 
Fo example value of EXTRA_LARGE is "Extra Large". `fromValue` should only assert equality to find the corresponding enum that holds that value, not change the input.
Iterate through the enums and return the first that has the same value.

Reply:
Should request than accept only "Extra Large" or "exTRA LaRge" as well?